base/gmp.jl

```diff
@@ -113,56 +113,54 @@ module MPZ
 #   and `add!(x, a) = add!(x, x, a)`.
 using Base.GMP: BigInt, Limb
 
-const mpz_t = Ref{BigInt}
```

Comment:
Why do you remove `mpz_t`? I thought it helps reading tedious libgmp signatures, and comparing them to the GMP docs (which use `mpz_t`).